resources/views/docs/1/the-basics/screens.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Screens
+order: 850
+---
+# Screens
+
+The `Screen` facade lets you get information about the screens currently connected to the computer.
+
+```php
+use Native\Laravel\Facades\Screen;
+```
+
+## Displays
+
+The `displays` method gives you an array of information about all the displays actually used.
+
+The Display object represents a physical display connected to the system. A fake display may exist on a headless system, or a display may correspond to a remote, virtual display. If you use an external display with your laptop screen closed, the internal screen of your laptop will not be part of the array.
+
+See [Display object](https://www.electronjs.org/docs/latest/api/structures/display) documentation.
+
+```php
+$screens = Screen::displays();
```

Comment:
Perhaps add a sample of the output here?

Reply:
It's a pretty massive array that's why I preferred linking to Electron. I can still put a var_export() example.

Reply:
👍🏼

Or just the relevant parts that help you demonstrate the example, whilst indicating there's more to discover, eg:

```php
// [
//     'relevantKeys' => 'exampleValue',
//     ...
// ]
```